backend/src/main/java/team31/codesuqad/sidedish/domain/Image.java

```diff
@@ -1,31 +1,34 @@
 package team31.codesuqad.sidedish.domain;
 
 import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+import team31.codesuqad.sidedish.type.ImageType;
 
 import java.util.Objects;
 
-public class Images {
+@Table("images")
+public class Image {
 
     @Id
     private Long imageId;
-    private String type;
-    private String url;
 
-    public Images(Long imageId, String type, String url) {
-        this.imageId = imageId;
+    private final ImageType type;
+    private final String url;
+
+    public Image(ImageType type, String url) {
         this.type = type;
         this.url = url;
     }
 
-    public boolean isSameType(String type) {
+    public boolean isSameType(ImageType type) {
         return Objects.equals(this.type, type);
     }
```

Comment:
대표이미지 검증용으로 사용되는 메소드 같네요.
아래와 같은 메소드로도 사용 가능할것같습니다 :)
```suggestion
    public boolean isMainType() {
        return this.type == ImageType.main;
    }
```